releasenotes/notes/added-runtime-params-class-a458e64bea4ec8f6.yaml

```
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    The `inputs` parameter to 
+    :meth:`qiskit.providers.ibmq.runtime.ibmq_runtime_service.run` 
+    method can now be specified as a
+    :class:`qiskit.providers.ibmq.runtime.runtime_program.ParameterNamespace`
+    to support auto-complete features.
```

```suggestion
features:
  - |
    The `inputs` parameter to
    :meth:`qiskit.providers.ibmq.runtime.IBMRuntimeService.run`
    method can now be specified as a
    :class:`qiskit.providers.ibmq.runtime.ParameterNamespace` instance which
    supports auto-complete features. You can use
    :meth:`qiskit.providers.ibmq.runtime.RuntimeProgram.parameters` to retrieve
    an ``ParameterNamespace`` instance.
upgrade:
  - |
    :meth:`qiskit.providers.ibmq.runtime.RuntimeProgram.parameters` is now a 
    method that returns a
    :class:`qiskit.providers.ibmq.runtime.ParameterNamespace` instance, which
    you can use to fill in runtime program parameter values and pass to
    :meth:`qiskit.providers.ibmq.runtime.IBMRuntimeService.run`.
    
```
We need both `upgrade` and `feature` because it's both an API change and new feature.

---

Fixed!